test/mbr_update_test.rb

```diff
@@ -28,6 +31,21 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:s390).and_return(true)
+      end
+
+      it "does nothing except returning true" do
+        expect(Yast::WFM).to_not receive(:Execute)
+        expect(Yast::SCR).to_not receive(:Execute)
+        expect_any_instance_of(::Bootloader::BootRecordBackup).to_not(
+          receive(:write)
+        )
+        expect(subject.run).to eq true
```

Comment:
I'm still not 100% satisfied, since there are other non-excluded reasons for WFM and SCR to not be  called (like ```Yast::BootCommon.globals["generic_mbr"]``` being "false" or `Yast::BootCommon.backup_mbr` being false), but I can live with it. So let's move forward.